digitaltwin_client/src/dt_version.c

```diff
@@ -3,6 +3,9 @@
 
 #include "digitaltwin_client_version.h"
 
+/** @brief Pre-processor friendly representation of Digital Twin Client version. */
```

Comment:
Please remove this comment as it's for Doxygen / external facing docs, which doesn't apply as this is internal facing now.